modules/nodes-descheduler-configuring-profiles.adoc

```diff
@@ -36,17 +36,19 @@ spec:
   managementState: Managed
   operatorLogLevel: Normal
   profileCustomizations:
-    podLifetime: 48h        <1>
-  profiles:                 <2>
+    podLifetime: 48h                                   <1>
+    thresholdPriorityClassName: my-priority-class-name <2>
```

Comment:
@bergerhoffer should we also give an example to set thresholdPriority / include & exclude namespaces ?

Reply:
@kasturinarra I can definitely add an example for namespaces to exclude. But I already have thresholdPriorityClassName set, and I believe you can only set either that OR thresholdPriority. So I just did the one here. Let me know if that's okay!

Reply:
@bergerhoffer that should be fine, but i was just wondering if a customer who would want to setup thersholdPriority in their cluster configuration may not find how to do that and this may not be super friendly, may be we could add another example and add only thersholdPriority if that is feasible  may be ?

Reply:
@bergerhoffer thanks, i think that looks fine to me !!

Reply:
@kasturinarra Thanks! I made that update, can you let me know if this PR is good to go now?

http://file.rdu.redhat.com/~ahoffer/2022/OSDOCS-3678/nodes/scheduling/nodes-descheduler.html#nodes-descheduler-configuring-profiles_nodes-descheduler

Reply:
@kasturinarra Okay added the namespaces to the example - ready for final review for real now :slightly_smiling_face: